blockmanager.go

```diff
@@ -251,13 +228,15 @@ func newBlockManager(cfg *blockManagerCfg) (*blockManager, error) {
 	}
 	bm.genesisHeader = *genesisHeader
 
+	bhdrProcessor := cfg.blkHdrProcessor
 	// Initialize the next checkpoint based on the current height.
-	header, height, err := cfg.BlockHeaders.ChainTip()
+	header, height, err := bhdrProcessor.store.ChainTip()
```

Comment:
Can have it be a direct public method instead of needing to reach into the attributes.

Reply:
Ohh okay thanks a lot.